pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaMPI.hpp

```diff
@@ -119,6 +119,7 @@ class StateVectorCudaMPI final
               handle_.get(), mpi_manager_, mpi_buf_size, BaseType::getData(),
               num_local_qubits, localStream_.get())),
           gate_cache_(true, dev_tag) {
+        BaseType::initSV();
```

Comment:
Is there any concern with the overhead of sero-state initialization in the construction? It seems a bit redundant.

Reply:
As discussed, let's get this support when we update `cudaMalloc` with `cudaMemset` later.